src/RealtimeChannel.ts

```diff
@@ -406,13 +406,21 @@ export default class RealtimeChannel {
   ): RealtimeChannel {
     return this._on(type, filter, callback)
   }
-
+  /**
+   * Sends a message into the channel.
+   *
+   * @param args Arguments to send to channel
+   * @param args.type One of "broadcast", "presence", or "postgres_changes".
```

Comment:
If type can only be those the values, you should type it, so it will also only show those 3 values when someone calls it

`type: string` => `type: 'broadcast' | 'presence' | 'postgres_change'`